src/python_redlines/api.py

```
@@ -0,0 +1,126 @@
+    """
+    Verifies the API token provided in the request header.
+    """
+    if not secrets.compare_digest(x_api_token, api_token):
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED, detail="Invalid API Token"
+        )
+
+
+@app.post("/redline/single", dependencies=[Depends(verify_token)])
+async def redline_single(
+    original: UploadFile = File(...),
```

Ruff (0.13.3)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)